setup.cfg

```diff
@@ -19,4 +19,3 @@ filterwarnings =
 
 [pytype]
 inputs = stable_baselines
-python_version = 3.5
```

Comment:
Are we dropping support for 3.5? If so we should specify the lowest supported version here. We should probably also add a `python_requires` in `setup.py`

Reply:
Is there a way to allow using pytype with python > 3.5?
I removed this line because I had to comment it each time I wanted to check typing, which was quite annoying.
If there is no other mean, I'll put that line back.

Reply:
so, no drop of python 3.5 support yet (I would have created an issue if this was the case)

Reply:
I think `pytype` always type checks with the Python version of the interpreter it's running under, which is a bit annoying. If you don't specify it then it'll use the system default. It seems unfortunate to get different type check results depending on what system you're running on, but uncommenting the line does sound frustrating.

Could you install Python 3.5 in addition to the Python interpreter you normally use?

Reply:
I can but this would mean duplicating a complete conda env only for this particular purpose.
i'll put the line back